networkx/algorithms/centrality/harmonic.py

```diff
@@ -6,7 +6,7 @@
 __all__ = ["harmonic_centrality"]
 
 
-def harmonic_centrality(G, nbunch=None, distance=None):
+def harmonic_centrality(G, nbunch=None, sources=None, distance=None):
```

Comment:
```suggestion
def harmonic_centrality(G, nbunch=None, distance=None, source=None):
```
Adding the new `sources` kwarg to the end avoids breaking the current API.